library/src/main/java/com/vinted/coper/CoperImpl.kt

```diff
@@ -59,6 +72,52 @@ internal class CoperImpl(private val fragmentManager: FragmentManager) : Coper {
         }
     }
 
+    internal suspend fun getFragmentSafely(): CoperFragment {
+        val deferred = CompletableDeferred<CoperFragment>()
```

Comment:
Also, the best way and idiomatic one is to use `suspendCoroutine`

Reply:
In suspendCoroutine, you cant call suspend functions

Reply:
Why do you need a call to suspend functions? Something seems wrong with code structure.

Reply:
Now i done it correctly, I was making all function suspending, when I would need a small protion of code just to make suspending, thats is bad that methods are too big

Reply:
So it means your structure is bad. It is a good chance some problems are hidden in your structure.

Reply:
I already fixed that, but I saw that too